index.js

```diff
@@ -59,6 +61,15 @@ async function fastifyStatic (fastify, opts) {
         : prefix + '/'
   }
 
+  if (opts.hash === true) {
+    if (opts.wildcard === undefined || opts.wildcard === true) {
```

Comment:
I'm not sure I like the DX of having to set wildcard explictly to `false`. If left undefined I think it should simply be overwritten?

Reply:
Sure, it was to prevent an unexpected scenario from happening where the dev would enable the feature and modify the location of a file while the server is running

Since wildcard is off, it won’t be updated and it could be a hard to figure out why without explicitly setting wildcard to false